tests/LPs/chat-feature.test.js

```diff
@@ -0,0 +1,49 @@
+
+const { test, expect } = require('@playwright/test');
+
+//Config
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://meet.swordhealth.com/');
+});
+
+//Main test: Chat feature  
+test.describe('Chat feature', () => {
+
+  //Test 1: Confirm element inside chat feature
+  test('When user clicks button, element should display', async ({ page }) => {
+    
+    // User clicks on the Chat button
+    await page.getByTestId('launcher').click();
+
+    // 2 seconds wait to open modal
+    await page.waitForTimeout(2000);
+
+    // User confirms element is visible on chat form
+    await expect(page.getByTestId('button-ok')).toBeVisible();
+    })
+
+
+
+  //Test 2: Fill in form and send 
+  test('User fills in the form', async ({ page }) => {
+
+    const form = page.getByTestId('forms.input');
+    
+    // User clicks on the Chat button
+    await page.getByTestId('launcher').click();
+
+    // 2 seconds wait to open modal
+    await page.waitForTimeout(2000);
+
+    // User fill in the form
+    await form[0].click().fill('user');
+    await form[1].click().fill('user@qa.com');
+    await form[2].click().fill('test');
+
+    // User send the form
+    await page.getByTestId('button-ok').click();
+
+    // Confirmation step
+    await expect(page.getByTestId('buttons.button')).toBeVisible();
```

Comment:
```suggestion
    // Chat is in a different frame than the button
    // Locate the chat frame
    const chatFrame =  await page.frameLocator('#webWidget');

    // Locate each form field by their name. It's a stable way to locate them and fill in the form
    await chatFrame.locator('[name="name"]').fill( 'user');
    await chatFrame.locator('[name="email"]').fill('user@test.com');
    await chatFrame.locator('[name="message"]').fill('test message');

    // User send the form
    await chatFrame.getByTestId('button-ok').click();

    // Locate the chat footer
    // This might need a condition if the chat is not open
    await expect(chatFrame.getByTestId('chat-footer-desktop')).toBeVisible();
```